getting-started.md

```diff
@@ -83,6 +83,54 @@ If everything works, you should see your starter app on your device or simulator
   </div>
 </div>
 
+## _Hot Reloading_ and _Full Application Restart_ from within IntelliJ
+
+To edit your code and hot reload changes:
+
+1. Run your app in debug mode.
+1. Make desired changes in your source code (without stopping the debugger).
+1. In the Debug panel at the bottom, click the Hot Reload button.
+
+![Hot reload in IntelliJ](/images/intellij/hot-reload.gif)
+
+Hot-reload works near instantly, offering a best-in-class development cycle.
+
+Not all source code changes can be hot-reloaded. To fully restart the Dart VM of
+your Flutter app you can use the _Full Application Restart_ button.
+
+![Main IntelliJ toolbar](/images/intellij/debug-toolbar.png)
+
+The _Rerun ``main.dart``_ button on the left side of the Debug Toolbar will
+redeploy the app to the device. For both _Full Application Restart_ and _Rerun_
+you will lose the state of your application.
+
+### IntelliJ Key Mappings _Hot Reloading_ and _Full Application Restart_
```

Comment:
We generally use the term 'hot reload' not 'hot reloading'. Please change throughout.